docs/book/src/user/concepts.md

```diff
@@ -29,13 +29,13 @@ The Bootstrap Provider is responsible for:
 
 The control plane is a set of [services](https://kubernetes.io/docs/concepts/#kubernetes-control-plane) that serve the Kubernetes API and continuously reconcile desired state using control loops.
 
-* __Machine-based__ control planes are the most common type, used by tools like kubeadm and kubespray. Dedicated machines are provisioned, running [static pods](https://kubernetes.io/docs/tasks/configure-pod-container/static-pod/) for components such as [kube-apiserver](https://kubernetes.io/docs/admin/kube-apiserver/), [kube-controller-manager](https://kubernetes.io/docs/admin/kube-controller-manager/) and [kube-scheduler](https://kubernetes.io/docs/admin/kube-scheduler/).
+* __Machine-based__ control planes are the most common type. Dedicated machines are provisioned, running [static pods](https://kubernetes.io/docs/tasks/configure-pod-container/static-pod/) for components such as [kube-apiserver](https://kubernetes.io/docs/admin/kube-apiserver/), [kube-controller-manager](https://kubernetes.io/docs/admin/kube-controller-manager/) and [kube-scheduler](https://kubernetes.io/docs/admin/kube-scheduler/).
 
 * __Pod-based__ deployments require an external hosting cluster. The control plane components are deployed using standard *Deployment* and *StatefulSet* objects and the API is exposed using a *Service*.
 
 * __External__ control planes are offered and controlled by some system other than Cluster API, such as GKE, AKS, EKS, or IKS.
 
-As of v1alpha2, __Machine-Based__ is the only control plane type that Cluster API supports.
+As of v1alpha2, __Machine-Based__ is the only control plane type that Cluster API supports. The default provider uses kubeadm to bootstrap the control plane. It exposes the configuration via the `kubeadmcontrolplane` object. The controller, `capi-kubeadm-control-plane-controller-manager`, can then create Machine and BootstrapConfig objects based on the requested replicas in the `kubeadmcontrolplanes` object.
```

Comment:
Only one minor concern given that KCP was introduced in v1alpha3, but given the initial sentence, the user might be tricked in assuming it should be there in v1alpha2 as well...

Reply:
good point :+1:, fixing.